site/content/docs/03-client-api.md

```diff
@@ -27,7 +27,9 @@ sapper.start({
 ### goto(href, options?)
 
 * `href` — the page to go to
-* `options` — can include a `replaceState` property, which determines whether to use `history.pushState` (the default) or `history.replaceState`. Not required
+* `options` - not required
```

Comment:
the dash on this line was changed so that it no longer matches the others on the page